test/azure/Expected/AcceptanceTests/AzureBodyDuration/bodyduration/aio/_configuration_async.py

```diff
@@ -32,7 +33,7 @@ def __init__(self, credentials, **kwargs):
         super(AutoRestDurationTestServiceConfiguration, self).__init__(**kwargs)
         self._configure(**kwargs)
 
-        self.user_agent_policy.add_user_agent('azsdk-python-autorestdurationtestservice/{}'.format(VERSION))
+        self.user_agent_policy.add_user_agent('autorestdurationtestservice/{}'.format(VERSION))
```

Comment:
Was this intentional?

Reply:
Found it, I was calling the vanilla template and not the azure one, and the vanilla aren't patched. I'm fixing two things then:
- Patching the vanilla with the right UserAgent
- Fixing this file generation to call the Azure one